Formula/libevent.rb

```diff
@@ -11,22 +11,32 @@ class Libevent < Formula
     sha256 "ef703db1b4cbdab35b89aabe80c225dd9b7a2c3ea14b1eae681478c5b9df15fe" => :yosemite
   end
 
-  depends_on "autoconf" => :build
-  depends_on "automake" => :build
+  head do
+    url "https://github.com/libevent/libevent", :using => :git
```

Comment:
`https://github.com/libevent/libevent.git` instead of `:using => :git` and there should be a blank line under this.